practices/graded-browser-support.md

```diff
@@ -83,6 +83,16 @@ Unfortunately, both IE10 and IE11 support the same CSS media queries, which mean
 
 This means that even if we consider IE10 a Grade-C browser, we currently serve IE10 users the full experience.
 
+#### Internet Explorer and CSS Grid
+
+[CSS Grid](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout) has been availalable in all evergreen browsers [for the last year](https://caniuse.com/#feat=css-grid). It offers a much-simplified way of working with layout and offers new design opportunities.
+
+However, CSS Grid is not available in the full standardised form in Internet Explorer 10 or 11, both of which are currently *de facto* supported by us (see above).
+
+The syntax and capabilities for CSS Grid are radically different in IE than it is in any other browser. As a result the development costs to match IE with other browsers can range from trivial, for simple localised layouts, to major for large-scale page layouts.
+
+We therefore accept that some teams will wish to develop their layouts using CSS Grid, but will not be able to justify devoting time to supporting IE. For those teams we suggest a highly-styled linear layout, which has all the features of a Grade A browser experience, except for the page layout. It is up to any teams making this decision to ensure the Grade A experience is acceptable in IE 10 and 11.
```

Comment:
acceptable to whom? also `It is up to any teams` -> `It's up to any teams` -- house style, contractions. I know this because @hollsk  made me fix about 20 million examples of this earlier in the week.

Reply:
lol sorry you're still salty 😘 

but while we're on the subject! `"will not be able"` -> `"won't be able"`

Reply:
I fear the wrath of @hollsk so I'll fix those. 😂